test_stack.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+
+from stack import Stack
+
+
+def test_stack():
+    s = Stack([3, 6, 2])
+
+    assert s.storage._length == 3
+    assert s.storage.head.value == 2
```

Comment:
You're technically testing the internal structure of the linked list here.  This ties your test to the specific way the LinkedList was written even though you don't need to care about how it was written.  When you do this you are more likely to break during Major version increments.  You can be confident that a LinkedList will always have the methods you need here, push, pop, and __len__. You can't be sure that it will always have a property .head, and contain Nodes with .value.  Since you don't ever use these in your code, avoid using them in your tests.